packages/cli-repl/src/cli-repl.ts

```diff
@@ -266,7 +288,7 @@ class CliRepl {
 
     try {
       fd = fs.openSync(configPath, 'wx');
-      this.userId = new bson.ObjectId(Date.now()).toString();
+      this.userId = new bson.ObjectId().toString();
```

Comment:
`Date.now()` returns ms, but `bson.ObjectId()` expects seconds, which is why this started to throw an overflow error in bson 4.2.0 … probably not intended on their side, but a bug on ours, and the `ObjectId` constructor defaults to the current timestamp anyway